### DISCUSSION TITLE
Include Sha256 Installer Hashes in the releases.

### DISCUSSION THREAD

**Discussion by ghost** [tnodir/fort#194] [Ideas]
This is a good idea and common to see around releases, just to make sure any Fort's installer matches the one being build by the dev, especially if the installers are being distributed outside Github.

**Comment by tnodir** [comment]
Installers are signed by my cert.

**Reply by ghost** [reply]
But what if someone else built it or repackaged, not everyone will download from Github and not everyone is going to check sources, of course, we have to assume if they do that, then nothing will protect them.
Integrity checks also would detect if the installer is corrupted or something like that, so it is not necessarily about malicious behavior, so just something to compare.

In theory it shouldn't matter, but SHA256 being added to many projects release notes inside and outside Github seems common, more and easier to see information the better?

**Reply by tnodir** [reply]
Github releases will contain links to Virustotal check results with SHA256.

**Reply by ghost** [reply]
Ohhh Amazing! thanks so much! This also will help to update Winget manifests faster, without having to download to generate the hash.
